ci/check-lint.sh

```diff
@@ -5,6 +5,8 @@ set -x
 CLIPPY() {
 	# shellcheck disable=SC2086
 	RUSTFLAGS='-D warnings' cargo clippy $1 -- $2 \
+		`# https://github.com/rust-lang/rust-clippy/issues/15442` \
+		-A clippy::unused_imports \
```

Comment:
```suggestion
		-A unused_imports \
```

It's actually a rustc lint. I left a comment on the linked issue with a possibly related issue in the main rust project.